examples/react-gqty/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "gqty",
```

```suggestion
  "name": "@nhost-examples/react-gqty",
```
turborepo relies on consistent names